src/setup/src/setup_httpd.erl

```diff
@@ -31,24 +31,30 @@ handle_setup_req(#httpd{method='GET'}=Req) ->
     ok = chttpd:verify_is_server_admin(Req),
     Dbs = chttpd:qs_json_value(Req, "ensure_dbs_exist", setup:cluster_system_dbs()),
     couch_log:notice("Dbs: ~p~n", [Dbs]),
-    case erlang:list_to_integer(config:get("cluster", "n", undefined)) of
-        1 ->
-            case setup:is_single_node_enabled(Dbs) of
-                false ->
-                    chttpd:send_json(Req, 200, {[{state, single_node_disabled}]});
-                true ->
-                    chttpd:send_json(Req, 200, {[{state, single_node_enabled}]})
-            end;
+    SingleNodeConfig = config:get_boolean("couchdb", "single_node", false),
+    case SingleNodeConfig of
+        true ->
+            chttpd:send_json(Req, 200, {[{state, single_node_enabled}]});
```

Comment:
`setup:is_single_node_enabled(Dbs)` isn't necessary?

Reply:
@gesellix yeah it’s a tiny bit of a fudge, I hoped we can get away with it :)

Reply:
if this ends up causing an issue in practice, we can add another clause there, but then we also need to invent a new state like `single_node_pending` or something and I didn’t want to get this in this late in the game.